.gitignore

```diff
@@ -122,7 +122,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
-.env
+aireview/src/aireview/.env
 .venv
```

Comment:
**style**: Replacing generic '.env' with specific path 'aireview/src/aireview/.env' reduces security by potentially exposing environment variables. Keep the generic '.env' pattern to ensure all .env files are ignored regardless of location.